File: Estruturas e Decisões/Desafio.py
nome = input("Digite o nome do produto: ")
quantidade = int(input("Digite a quantidade em estoque: "))
preco = float(input("Digite o preço unitário: "))

if quantidade >= 0:
    print("\n--- Produto Cadastrado ---")
    print("Nome:", nome)
    print("Quantidade:", quantidade)
    print("Preço:", preco)
else:
    print("Erro: A quantidade não pode ser um valor negativo.")

File: Estruturas e Decisões/Desafio.cpp
#include <iostream>
using namespace std;

int main() {
    string nome;
    int quantidade;
    double preco;

    cout << "Digite o nome do produto: ";
    getline(cin, nome);

    cout << "Digite a quantidade em estoque: ";
    cin >> quantidade;

    cout << "Digite o preço unitário: ";
    cin >> preco;

    if (quantidade >= 0) {
        cout << "\n--- Produto Cadastrado ---" << endl;
        cout << "Nome: " << nome << endl;
        cout << "Quantidade: " << quantidade << endl;
        cout << "Preço: " << preco << endl;
    } else {
        cout << "Erro: A quantidade não pode ser um valor negativo." << endl;
    }

    return 0;
}